src/Github.php

```diff
@@ -177,6 +179,22 @@ public function setFullTime($value)
         return $this->updateQuery($fullTime, 'full_time');
     }
 
+    public function setCityStateLocation($job)
+    {
+        if (isset($job->location)) {
```

Comment:
Not sure I understand. The location attribute is on the job object. Do you mean I should use the acceser (getLocation) rather than accessing the attribute directly?